website/blog/2020-02-26-cobalt.md

```diff
@@ -0,0 +1,449 @@
+---
+author: Tomasz Godzik
+title: Metals v0.8.1 - Cobalt
+authorURL: https://twitter.com/TomekGodzik
+authorImageURL: https://github.com/tgodzik.png
+---
+
+We are excited to announce the release of Metals v0.8.1. This release includes a
+large number of bug fixes and some new features.
+
+<table>
+<tbody>
+  <tr>
+    <td>Commits since last release</td>
+    <td align="center">313</td>
+  </tr>
+  <tr>
+    <td>Merged PRs</td>
+    <td align="center">100</td>
+  </tr>
+    <tr>
+    <td>Contributors</td>
+    <td align="center">19</td>
+  </tr>
+  <tr>
+    <td>Closed issues</td>
+    <td align="center">52</td>
+  </tr>
+  <tr>
+    <td>New features</td>
+    <td align="center">2</td>
+  </tr>
+</tbody>
+</table>
+
+For full details: https://github.com/scalameta/metals/milestone/19?closed=1
+
+Metals is a language server for Scala that works with VS Code, Vim, Emacs,
+Sublime Text, Atom and Eclipse. Metals is developed at the
+[Scala Center](https://scala.epfl.ch/) and [VirtusLab](https://virtuslab.com)
+with help from [Lunatech](https://lunatech.com) along with contributors from the
+community.
+
+## TL;DR
+
+Check out [https://scalameta.org/metals/](https://scalameta.org/metals/) and
+give Metals a try!
+
+- insert missing abstract members
+- new file provider
+- enable rename preview for Visual Studio Code
+- debug and run working in Emacs
+
+## Insert missing abstract members
+
+Thanks to yet again amazing work by
+[tanishiking](https://github.com/tanishiking), we now support the quick fix code
+action for implementing abstract class members. This code action is available in
+case of errors and works in the same way as the recent "Implement all members"
```

Comment:
It might be helpful to add a link to https://scalameta.org/metals/blog/#completion-to-add-all-abstract-members on `recent "Implement all members" completion` for people who are not familiar with the 0.8.0 release.

Reply:
Good idea! Thanks!